File: Aulas LP3(Codigos em Java, JavaScript)/Pessoa.java
import java.time.LocalDate;

public class Pessoa {

    // ATRIBUTOS

    private String nome;
    private LocalDate nascimento;


//MÉTODOS CONSTRUTORES

    public Pessoa(){
        this.nome = "";
        this.nascimento = LocalDate.now();
    }
    public Pessoa (String nome, Integer dia, Integer mes, Integer ano){
        this.nome = nome;
        this.nascimento = LocalDate.of(ano, mes, dia);
    }
    

//MÉTODOS GET E SET

public String getNome(){
    return this.nome;
}
public void setNome(String nome){
    this.nome = nome;
}

public LocalDate getNascimento(){
    return nascimento;
}

//MÉTODO ToSring

public String toString(){
    return "[nome: "+nome+ " nascimento: "+ nascimento+"]";
}

}

File: Aulas LP3(Códigos em C#)/Estudo para prova/classes/Pessoa.cs
namespace classes
{
    
public class Pessoa{
    private string nome;
    private int idade;

    public Pessoa (string nome, int idade){
        this.nome=nome;
        this.idade=idade;
    }

    public string getNome(){
        return nome;
    }
    public void setNome(string nome){
        this.nome=nome;
    }

    public int getIdade(){
        return idade;
    }
    public void setIdade(int idade){
        this.idade=idade;
    }

    public string toString (){
//return ($"Nome: {nome}\n" + $" Idade: {idade}");
return ("Nome: "+ nome.ToUpper() + "  Idade: "+ idade);
    }


}
}
